account-kit/smart-contracts/src/ma-v2/client/client.test.ts

```diff
@@ -1,4 +1,10 @@
-import { custom, parseEther, publicActions, zeroAddress } from "viem";
+import {
+  custom,
+  getContract,
```

Comment:
⚠️ **[eslint]** <[@typescript-eslint/no-unused-vars](https://typescript-eslint.io/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'getContract' is defined but never used.
<!-- __reviewdog__:ChAyNGZiZWFiYzBkNWVlNDY0EgZyZGpzb24= -->